Makefile

```diff
@@ -47,6 +48,10 @@ bin_win:
 test:
 	go test -v ./...
 
+.PHONY: build
+build:
+	go build -ldflags "-X 'github.com/redhat-certification/chart-verifier/cmd.CommitIDLong=$(COMMIT_ID_LONG)'"
+
```

Comment:
Lets use the existing `bin` target instead of having a separate `build` target. Also add this to `bin_win` for now. 

I'll collapse those targets at some point in the future, but for now, we can duplicate the logic in both places.